modules/dev-tools/templates/.eslintrc.js

```diff
@@ -0,0 +1,59 @@
+const typescriptConfigs = require('@typescript-eslint/eslint-plugin').configs;
+
+module.exports = {
+  parser: 'babel-eslint',
+  parserOptions: {
+    ecmaVersion: 2018
+  },
+  plugins: ['markdown'],
+  extends: [
+    'uber-es2015',
+    'prettier'
+  ],
+  rules: {
+    'guard-for-in': 0,
+    'no-inline-comments': 0,
+    'camelcase': 0
+  },
+  overrides: [
+    {
+      files: ['**/*.ts', '**/*.tsx'],
+      parser: '@typescript-eslint/parser',
+      env: {browser: true, es6: true, node: true},
+      parserOptions: {
+        sourceType: 'module',
+        project: './tsconfig.json'
+      },
+      plugins: ['@typescript-eslint'],
+      rules: {
+        ...typescriptConfigs['eslint-recommended'].rules,
```

Comment:
Do you have any notes about which rules affect performance most. I still think performance is a bit of an issue with this plugin.

Reply:
Not really. I'd guess `recommended-requiring-type-checking` are probably the most expensive.